src/main/java/se/bjurr/violations/lib/example/MyClass.java

```diff
@@ -9,6 +9,8 @@ void npe(String a, String b) {
   } else {
```

Comment:
**Reporter**: FINDBUGS
**Rule**: NP_NULL_ON_SOME_PATH
**Severity**: ERROR
**File**: se/bjurr/violations/lib/example/MyClass.java L26
**Source**: se.bjurr.violations.lib.example.MyClass

Possible null pointer dereference&#10;&#10;&#10;&lt;p&gt; There is a branch of statement that, &lt;em&gt;if executed,&lt;/em&gt;  guarantees that&#10;a null value will be dereferenced, which&#10;would generate a &lt;code&gt;NullPointerException&lt;/code&gt; when the code is executed.&#10;Of course, the problem might be that the branch or statement is infeasible and that&#10;the null pointer exception can&#39;t ever be executed; deciding that is beyond the ability of FindBugs.&#10;&lt;/p&gt;&#10;


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1205985263>*